binary/grakn

```diff
@@ -74,9 +74,8 @@ elif [[ "$1" = "server" ]] || [[ "$1" = "version" ]]; then
         CLASSPATH="${GRAKN_HOME}/server/conf/:${GRAKN_HOME}/${LIB_COMMON}"
         # exec replaces current shell process with java so no commands after these ones will ever get executed
         if [ $DEBUG ]; then
-            echo "Running Grakn Core Server in debug mode."
             CLASSPATH="${CLASSPATH}:${GRAKN_HOME}/server/lib/dev/*"
-            exec java ${GRAKN_DAEMON_JAVAOPTS} -ea -cp "${CLASSPATH}" -Dgrakn.dir="${GRAKN_HOME}" -Dgrakn.conf="${GRAKN_HOME}/${GRAKN_CONFIG}" -Dstorage.javaopts="${STORAGE_JAVAOPTS}" -Dserver.javaopts="${SERVER_JAVAOPTS}" grakn.core.server.GraknServer "${@:2}"
+            exec java ${GRAKN_DAEMON_JAVAOPTS} -ea -cp "${CLASSPATH}" -Dgrakn.dir="${GRAKN_HOME}" -Dgrakn.conf="${GRAKN_HOME}/${GRAKN_CONFIG}" -Dstorage.javaopts="${STORAGE_JAVAOPTS}" -Dserver.javaopts="${SERVER_JAVAOPTS}" -Dlogback.configurationFile=logback-debug.xml grakn.core.server.GraknServer "${@:2}"
```

Comment:
what about this:

```
        CLASSPATH=":${GRAKN_HOME}/${LIB_COMMON}"
        # exec replaces current shell process with java so no commands after these ones will ever get executed
        if [ $DEBUG ]; then
            echo "Running Grakn Core Server in debug mode."
            CLASSPATH="${CLASSPATH}:${GRAKN_HOME}/server/conf/logback-debug.xml:${GRAKN_HOME}/server/lib/dev/*"
            exec java ${GRAKN_DAEMON_JAVAOPTS} -ea -cp "${CLASSPATH}" -Dgrakn.dir="${GRAKN_HOME}" -Dgrakn.conf="${GRAKN_HOME}/${GRAKN_CONFIG}" -Dstorage.javaopts="${STORAGE_JAVAOPTS}" -Dserver.javaopts="${SERVER_JAVAOPTS}" grakn.core.server.GraknServer "${@:2}"
            exec java ${GRAKN_DAEMON_JAVAOPTS} -ea -cp "${CLASSPATH}" -Dgrakn.dir="${GRAKN_HOME}" -Dgrakn.conf="${GRAKN_HOME}/${GRAKN_CONFIG}" -Dstorage.javaopts="${STORAGE_JAVAOPTS}" -Dserver.javaopts="${SERVER_JAVAOPTS}" grakn.core.server.GraknServer "${@:2}"
        else
            CLASSPATH="${CLASSPATH}:${GRAKN_HOME}/server/conf/logback.xml:${GRAKN_HOME}/server/lib/prod/*"
            exec java ${GRAKN_DAEMON_JAVAOPTS} -cp "${CLASSPATH}" -Dgrakn.dir="${GRAKN_HOME}" -Dgrakn.conf="${GRAKN_HOME}/${GRAKN_CONFIG}" -Dstorage.javaopts="${STORAGE_JAVAOPTS}" -Dserver.javaopts="${SERVER_JAVAOPTS}" grakn.core.server.GraknServer "${@:2}"
        fi
```

Reply:
what about we select the logback file in the same way as how we select rocksdbjni?

i feel like it's nicer this way compared to adding a new `-Dlogback.configurationFille`, because:
 - we minimise the number of configurations
 - we reuse an existing approach that we've used for rocksdbjni anyway

Reply:
That doesn't work - what that change is modifying is the Java `classpath` which is used for Java sources (JARs, generally).

`-Dlogback.configurationFile` appears to be the standard way to use an alternate config file in logback.